lib/client/profile.go

```diff
@@ -242,6 +242,9 @@ type ProfileStatus struct {
 	// SAMLSingleLogoutEnabled is whether SAML SLO (single logout) is enabled, this can only be true if this is a SAML SSO session
 	// using an auth connector with a SAML SLO URL configured.
 	SAMLSingleLogoutEnabled bool
+
+	// SSOHostname is the hostname of the SSO provider used to login..
+	SSOHostname string
```

Comment:
Oh my bad, I was under the impression that `proxy_host` is currently returned specifically only from `GetCluster` and not from `ListRootClusters`. But it's actually included in `ListRootClusters` as well, but only for clusters to which the user has successfully logged in at least once. That's because it's also stored on the profile.

My thinking was that if `ssoHostname` is not something we absolutely need to have on the profile, we can just include it in the response we get from `GetCluster`.

We might be better off storing this in the profile though. This guarantees that Connect always has this piece of data available once the user logs in to a cluster. If `ssoHostname` was included only in `GetCluster`, then it'd be possible to launch Connect while your cert was still valid and perform an action that requires MFA before the app gets a response from `GetCluster`.

If something is in response from `ListRootClusters`, it's guaranteed to be present in the Electron app before the user has a chance to perform any action.

Reply:
Should we document that in the profile so that it's more clear why `SSOHostname` is in the profile? Otherwise people might wonder why it's there in the first place, since AFAIK it's useful only for Connect right now.